packages/stream_chat_flutter/lib/src/user_list_view.dart

```diff
@@ -194,12 +194,10 @@ class _UserListViewState extends State<UserListView>
   bool get isListAlreadySorted =>
       widget.sort?.any((e) => e.field == 'name' && e.direction == 1) ?? false;
 
-  Widget _buildError(Error error) {
-    print(error.stackTrace);
-
+  Widget _buildError(Object error) {
     var message = error.toString();
     if (error is DioError) {
-      final dioError = error as DioError;
+      final dioError = error;
```

Comment:
we don't need to check for dio errors now instead, check for `StreamChatError.isRetriable`.